<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-M01B47M" ignored="2" name="Test(failed)" tests="3" failures="2" timestamp="12 Jan 2023 15:28:53 GMT" time="31.213" errors="0">
  <testcase name="@BeforeMethod startUp" time="29.833" classname="basictestng.GoogleApplicationWithBeforeAfterMethod">
    <failure type="org.openqa.selenium.WebDriverException" message="Timed out waiting for driver server to start.
Build info: version: &amp;apos;3.141.59&amp;apos;, revision: &amp;apos;e82be7d358&amp;apos;, time: &amp;apos;2018-11-14T08:17:03&amp;apos;
System info: host: &amp;apos;DESKTOP-M01B47M&amp;apos;, ip: &amp;apos;192.168.43.164&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;1.8.0_341&amp;apos;
Driver info: driver.version: ChromeDriver">
      <![CDATA[org.openqa.selenium.WebDriverException: Timed out waiting for driver server to start.
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'DESKTOP-M01B47M', ip: '192.168.43.164', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_341'
Driver info: driver.version: ChromeDriver
at org.openqa.selenium.remote.service.DriverService.waitUntilAvailable(DriverService.java:202)
at org.openqa.selenium.remote.service.DriverService.start(DriverService.java:188)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:79)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:181)
at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:168)
at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:123)
at utilities2.SeleniumUtility.setUp(SeleniumUtility.java:45)
at basictestng.GoogleApplicationWithBeforeAfterMethod.startUp(GoogleApplicationWithBeforeAfterMethod.java:16)
Caused by: org.openqa.selenium.net.UrlChecker$TimeoutException: Timed out waiting for [http://localhost:2418/status] to be available after 20023 ms
at org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:100)
at org.openqa.selenium.remote.service.DriverService.waitUntilAvailable(DriverService.java:197)
... 38 more
Caused by: java.util.concurrent.TimeoutException
at java.util.concurrent.FutureTask.get(Unknown Source)
at com.google.common.util.concurrent.SimpleTimeLimiter.callWithTimeout(SimpleTimeLimiter.java:156)
at org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:75)
... 39 more
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod startUp -->
  <testcase name="@AfterMethod tearDown" time="0.0" classname="basictestng.GoogleApplicationWithBeforeAfterMethod">
    <skipped/>
  </testcase> <!-- @AfterMethod tearDown -->
  <testcase name="testSearch" time="0.0" classname="basictestng.GoogleApplicationWithBeforeAfterMethod">
    <skipped/>
  </testcase> <!-- testSearch -->
  <testcase name="testCase1" time="0.0" classname="basictestng.FirstTestNGWebScript">
    <failure type="java.lang.Error" message="Unresolved compilation problem: 
	The method cleanUp() is undefined for the type SeleniumUtility
">
      <![CDATA[java.lang.Error: Unresolved compilation problem: 
The method cleanUp() is undefined for the type SeleniumUtility

at basictestng.FirstTestNGWebScript.testCase1(FirstTestNGWebScript.java:31)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- testCase1 -->
  <testcase name="executeTimeout" time="1.018" classname="basictestng.Actitime_ExceptionAndTimeOut">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method basictestng.Actitime_ExceptionAndTimeOut.executeTimeout() didn&amp;apos;t finish within the time-out 1000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method basictestng.Actitime_ExceptionAndTimeOut.executeTimeout() didn't finish within the time-out 1000
at java.lang.Thread.interrupt(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 2 stack frames]]>
    </failure>
  </testcase> <!-- executeTimeout -->
  <testcase name="testGoogleLandingPage" time="0.0" classname="basictestng.GoogleApplicationWithBeforeAfterMethod">
    <ignored/>
  </testcase> <!-- testGoogleLandingPage -->
  <testcase name="divideByZeroExample" time="0.0" classname="basictestng.Actitime_ExceptionAndTimeOut">
    <ignored/>
  </testcase> <!-- divideByZeroExample -->
</testsuite> <!-- Test(failed) -->
